linux-cachyos/PKGBUILD

```diff
@@ -619,6 +625,12 @@ _package-headers() {
     pkgdesc="Headers and scripts for building modules for the $pkgdesc kernel"
     depends=('pahole' "${pkgbase}")
 
+    # Replace LTO kernel with the default kernel
+    if [[ "$_use_llvm_lto" = "thin" || "$_use_llvm_lto" = "full" ]] || [ -n "$_use_kcfi" ]; then
```

Comment:
Given the number of uses of this condition, it can be put into a separate function so that we can change the condition only once in case of anything. E. g;

```
_is_lto_kernel() {
    [[ "$_use_llvm_lto" = "thin" || "$_use_llvm_lto" = "full" ]] || [ -n "$_use_kcfi" ]
    return $?
}
```

Reply:
Let me know if I did anything wrong in the implementation.

Reply:
No, it's alright.